### ISSUE TITLE
[FEAT] Login Page

### ISSUE THREAD

**Issue by dimmujed** [linuxserver/emulatorjs#119]
### Is this a new feature request?

- [X] I have searched the existing issues

### Wanted change

A login page with some kind of basic authentication would be great for this, as that would allow it to be exposed to the internet w/o having to deal with copyright issues. I know there are ways to do it via other apps, but it seems like a rather complicated setup for one item.

### Reason for change

Privacy, plus you can make it so it auto sets the user profile with their saves.

**Comment by gr33k** [comment]
I wanted to do the same thing to protect my public facing site...but I've read that security is not tested on this thing as it is intended to be use on a LAN (fair enough). SOOoooo.....I want to share it only with my friends. Using NGINX I created a basic auth for this purpose and it works fine:

```
server {
    server_name your.website.com;

#Basic Aut
    auth_basic           "Cool Gamers Only";
    auth_basic_user_file /data/Containers/emulatorjs/nginx/.htpasswd;

    location / {
        proxy_pass http://192.168.0.10:80;
    }
```
Hope this helps!

**Comment by gr33k** [comment]
Also to note - this is a dupe of https://github.com/linuxserver/emulatorjs/issues/114

**Comment by thelamer** [comment]
Linking note https://github.com/linuxserver/emulatorjs/issues/114#issuecomment-1882239359

**Comment by thelamer** [comment]
I have decided against this, anyone with a home server should be putting this behind a reverse proxy of some kind. I am not security focused and any authentication layer added would be to keep the kids out not the internet. When people land on apps and have root level access they understand that they cannot expose them to the internet. When you add auth you are indirectly saying I have secured this application even if not communicating that directly.
